autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/obstacle_cruise_planner.param.yaml

```diff
@@ -12,8 +12,9 @@
       idling_time: 2.0 # idling time to detect front vehicle starting deceleration [s]
       min_ego_accel_for_rss: -1.0 # ego's acceleration to calculate RSS distance [m/ss]
       min_object_accel_for_rss: -1.0 # front obstacle's acceleration to calculate RSS distance [m/ss]
-      safe_distance_margin : 6.0 # This is also used as a stop margin [m]
-      terminal_safe_distance_margin : 3.0 # Stop margin at the goal. This value cannot exceed safe distance margin. [m]
+      cruise_safe_distance_margin : 0.5 # This is used as a cruise margin [m]
```

Comment:
Due to the discontinuous distance when switching from CRUISE to STOP or STOP to CRUISE, we came across the issue where the velocity is sometimes not comfortable in the experiment.
With the current logic, It would be better to set parameters to the same values by default. Is it okay for you?

Reply:
Thanks for your review @takayuki5168 -san.

> It would be better to set parameters to the same values by default.
Do you mean by this comment to set the default value of `cruise_safe_distance_margin `  with 6.0 meters by default ? 

if the answer is yes, so if this `cruise_safe_distance_margin ` a bit bigger value than 0.5, (2.0 for example) that would solve the mentioned problem ?

Reply:
Thanks again @takayuki5168 -san for your comment.

Actually having the value of `cruise_safe_distance_margin` with 6.0 will turn into that the intended Dense Urban ODD scenarios for which we are proposing this change, they will still be failing with main Autoware !!

In addition, what will be then the significance of the parameters separation in `obstacle_cruise_planner` package implemented using autoware.universe [PR](https://github.com/autowarefoundation/autoware.universe/pull/8166) if we are setting both parameters with the same value ? 
The purpose of separating these two parameters is having the ability to assign different values for them tackling different driving situation.

If we confirm that the `discontinuous distance when switching from CRUISE to STOP or STOP to CRUISE` is leading to some behavior degradation and failing of other scenarios, then it would be better that you share more details about this problem and maybe we can work on adding a solution for it.

Reply:
@ahmeddesokyebrahim 
Sorry, but I don't have a detailed scenario.
Theoretically, during the CRUISE state, the stable target distance is `cruise_safe_distance_margin`, and after switching from CRUISE to STOP, the target distance becomes `stop_safe_distance_margin`. Therefore, if `cruise_safe_distance_margin` is smaller than `stop_safe_distance_margin`, the ego cannot keep the distance `stop_safe_distance_margin` when the ego stops.
That's why these distances have to be the same.

Reply:
Thanks @takayuki5168 -san for your reply.

The development of this PR will be on-hold until further changes to be done in the scenario itself. 
Afterwords, we can re-test again and check if we still need the parameters separation or not, and based on that we can check the discontinuity problem.